sdk/ai/ai-voicelive/cspell.json

```diff
@@ -0,0 +1,6 @@
+{
+  // Version of the setting file.  Always 0.2
```

Comment:
We can move these to the centralized https://github.com/Azure/azure-sdk-for-js/blob/main/.vscode/cspell.json but filtered by package path instead of having another one